src/components/Dashboard/DashboardCallout.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { ReactNode } from "react";
+import {
+  Popup,
+  Button,
+  MoreIcon,
+  ComponentEventHandler,
+  PopupProps,
+  Menu,
+  mergeThemes,
+  ComponentVariablesInput,
+  Provider as FluentUIThemeProvider,
+  ThemePrepared,
+  EyeSlashIcon,
+} from "@fluentui/react-northstar";
+
+export interface IWidgetActionKey {
+  id: string;
+  icon?: JSX.Element;
+  title: string;
+}
+
+export interface IDashboardCallout {
+  open: boolean;
+  onOpenChange: ComponentEventHandler<PopupProps>;
+  menuProps: any;
+  globalTheme: ThemePrepared;
+  widgetActionGroup?: IWidgetActionKey[];
+}
+
+const getLocalTheme = () => {
+  return {
+    componentVariables: {
+      PopupContent: ({
```

Comment:
`PopupContent` isn't already defined in `DashboardTheme.tsx` — it's probably safe to remove the theming logic here and move it into `DashboardTheme`. Generally speaking I think we should probably avoid using nested local themes as it becomes hard to reason about, though I understand there's a caching argument for it…

Reply:
@thure I agree, I went with this way to prevent changes for all nested components that could be pass into card(widget) component.
If those style updates will be part of the parent theme component, all changes will be applied there as well.
That's why I have to do a separation to avoid any bugs in the future

Reply:
I see, so you're saying Cards might have a Popup, and these styles should not apply to those? Makes sense to me, actually.